src/tests/Loader/ContextualReflection/ContextualReflection.cs

```diff
@@ -751,8 +751,10 @@ public void RunTests()
             VerifyContextualReflectionProxy();
             VerifyUsingStatementContextualReflectionUsage();
             VerifyBadContextualReflectionUsage();
-            TestDynamicAssembly(true);
-            TestDynamicAssembly(false);
+
+            // TestDynamicAssembly() disabled due to https://github.com/dotnet/runtime/issues/48579
+            //TestDynamicAssembly(true);
```

Comment:
Are both the tests failing?

Reply:
Don't know. Some call to TestDefineDynamicAssembly fails. (Honestly, I don't care very much, since I'm ok with a "big hammer" if it's temporary, to get the CI running clean)